include/mbedtls/ssl.h

```diff
@@ -84,8 +84,13 @@
 #define MBEDTLS_ERR_SSL_BAD_CERTIFICATE                   -0x7A00
 /* Error space gap */
 /**
- * Received NewSessionTicket Post Handshake Message.
- * This error code is experimental and may be changed or removed without notice.
+ * A TLS 1.3 NewSessionTicket message has been received.
+ * This error code can be returned only on client side if and only if handling
+ * of TLS 1.3 NewSessionTicket messages has been enabled through the
+ * mbedtls_ssl_conf_enable_new_session_tickets() API. This error
+ * code can then be returned by mbedtls_ssl_handshake(),
+ * mbedtls_ssl_handshake_step(), mbedtls_ssl_read(), mbedtls_ssl_write() and
+ * mbedtls_ssl_write_early_data().
```

Comment:
We can't have long error descriptions because of a [limitation in `generate_errors.pl`: it grabs the whole thing](https://github.com/Mbed-TLS/mbedtls/issues/8783). (Preexisting problem for `MBEDTLS_ERR_SSL_CANNOT_READ_EARLY_DATA` by the way.)

Reply:
okay what is the usual way to deal with that then?

Reply:
The usual way is to write a simple sentence as the error description, and lobby our management for time to fix our technical debt.